tests/framework/matcher/phase.go

```diff
@@ -27,6 +27,16 @@ func HaveSucceeded() types.GomegaMatcher {
 	}
 }
 
+func PendingPopulation() types.GomegaMatcher {
+	return phaseMatcher{
+		expectedPhase: "PendingPopulation",
+	}
+}
+
+func WaitForFirstConsumer() types.GomegaMatcher {
+	return Or(BeInPhase("WaitForFirstConsumer"), PendingPopulation())
```

Comment:
could use `cdiv1.WaitForFirstConsumer`

Reply:
yeah I know but all other checks use the string and not the defined phase, didn't want to add another import just for that and I dont think it will change.